# Mastodon to Memos Localization Messages.
# Copyright (C) 2025 Pierre QUILLERY
# This file is distributed under the same license as the Mastodon2Memos package.
# Pierre <pierre@quillery.fr>, 2025.
#
msgid ""
msgstr ""
"Project-Id-Version: Mastodon2Memos 1.0\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2025-01-15 21:55+0100\n"
"PO-Revision-Date: 2025-01-15 21:44+0100\n"
"Last-Translator: Pierre Quillery <pierre@quillery.fr>\n"
"Language-Team: Spanish <es@tp.org.es>\n"
"Language: es\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: mastodon2memos/commands/__init__.py:16
#: mastodon2memos/commands/commands.py:16
#, python-brace-format
msgid ""
"Configuration file not found at {config_path}. Please ensure the file exists."
msgstr ""
"Archivo de configuración no encontrado en {config_path}. Por favor, "
"asegúrese de que el archivo existe."

#: mastodon2memos/commands/__init__.py:29
#: mastodon2memos/commands/commands.py:29
#, python-brace-format
msgid "Missing section \"{section}\" in configuration file."
msgstr "Falta la sección \"{section}\" en el archivo de configuración."

#: mastodon2memos/commands/__init__.py:33
#: mastodon2memos/commands/commands.py:33
#, python-brace-format
msgid "Missing key \"{key}\" in section \"{section}\" of configuration file."
msgstr ""
"Falta la clave \"{key}\" en la sección \"{section}\" del archivo de "
"configuración."

#: mastodon2memos/commands/__init__.py:60
#: mastodon2memos/commands/commands.py:60
msgid "Successfully connected to Mastodon API."
msgstr "Conexión exitosa a la API de Mastodon."

#: mastodon2memos/commands/__init__.py:62
#: mastodon2memos/commands/commands.py:62
msgid "Failed to connect to Mastodon API."
msgstr "Error al conectar con la API de Mastodon."

#: mastodon2memos/commands/__init__.py:69
#: mastodon2memos/commands/commands.py:69
msgid "Successfully connected to Memos API."
msgstr "Conexión exitosa a la API de Memos."

#: mastodon2memos/commands/__init__.py:71
#: mastodon2memos/commands/commands.py:71
msgid "Failed to connect to Memos API."
msgstr "Error al conectar con la API de Memos."

#: mastodon2memos/commands/mastodon2memos.py:6
msgid "Publish Mastodon toots as Memos."
msgstr "Publicar toots de Mastodon como Memos."

#: mastodon2memos/commands/mastodon2memos.py:7
msgid "Ask before creating a new memo from a toot."
msgstr "Preguntar antes de crear un nuevo memo a partir de un toot."

#: mastodon2memos/commands/mastodon2memos.py:8
#: mastodon2memos/commands/troubleshoot.py:7
msgid "Path to the configuration file."
msgstr "Ruta al archivo de configuración."

#: mastodon2memos/commands/mastodon2memos.py:25
#, python-brace-format
msgid "Processing toot: {url}"
msgstr "Procesando toot: {url}"

#: mastodon2memos/commands/mastodon2memos.py:28
#, python-brace-format
msgid ""
"Do you want to attempt publishing this toot as a memo?\n"
"{content}\n"
"{url}?"
msgstr ""
"¿Quieres intentar publicar este toot como un memo?\n"
"{content}\n"
"{url}?"

#: mastodon2memos/commands/mastodon2memos.py:29
msgid "Skipping toot."
msgstr "Saltando toot."

#: mastodon2memos/commands/mastodon2memos.py:33
msgid "New toot added to memos."
msgstr "Nuevo toot añadido a los memos."

#: mastodon2memos/commands/mastodon2memos.py:35
msgid "No memo added, toot was already saved in memos."
msgstr "No se añadió ningún memo, el toot ya estaba guardado en los memos."

#: mastodon2memos/commands/mastodon2memos.py:37
msgid "Finished processing toots."
msgstr "Procesamiento de toots finalizado."

#: mastodon2memos/commands/troubleshoot.py:6
msgid "Ensure proper connection to both Mastodon and Memos APIs."
msgstr ""
"Asegúrese de tener una conexión adecuada con las APIs de Mastodon y Memos."

#: mastodon2memos/commands/troubleshoot.py:20
#, python-brace-format
msgid "Successfully resolved Mastodon username to user ID: {user_id}"
msgstr ""
"Nombre de usuario de Mastodon resuelto con éxito a ID de usuario: {user_id}"

#: mastodon2memos/commands/troubleshoot.py:22
msgid "Failed to resolve Mastodon username to user ID."
msgstr "Error al resolver el nombre de usuario de Mastodon a ID de usuario."

#: mastodon2memos/clients/mastodon.py:51
#, python-brace-format
msgid "Failed to fetch latest public toots: {error}"
msgstr "Error al obtener los últimos toots públicos: {error}"

#: mastodon2memos/clients/memos.py:42
#, fuzzy, python-brace-format
msgid "> 🌐 [original toot]({url}) posted by [{acct}]({account_url}) #{tag}"
msgstr ""
"> 🌐 [toot original]({url}) publicado por [{acct}]({account_url}) #{tag}"
